docs/src/examples/components/Tooltip/Variations/index.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Variations = () => (
+  <ExampleSection title="Variations">
+    <ComponentExample
+      title="Alignment, Position and Pointing"
```

Comment:
```suggestion
      title="Alignment & Position"
```

I suggest to also add a separate example to `pointing` in `Types` section because it's a type